cluster/saltbase/salt/flannel-server/flannel-server.manifest

```diff
@@ -0,0 +1,99 @@
+{
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "flannel-helper",
+        "namespace": "kube-system",
+        "labels": {
+            "app": "flannel-helper",
+            "version": "v0.1"
+        }
+    },
+    "spec": {
+        "volumes": [
+            {
+                "name": "varlogflannel",
+                "hostPath": {
+                    "path": "/var/log"
+                }
+            },
+            {
+                "name": "etcdstorage",
+                "emptyDir": {}
+            },
+            {
+                "name": "networkconfig",
+                "hostPath": {
+                    "path": "/etc/kubernetes/network.json"
```

Comment:
Or we can only update it by restarting the pod right? which is what I'd expect (updating the network requires at least a kube-push). ideally we would store this in something like a config blob I guess, if not a network resource.